src/core/annotation.js

```diff
@@ -343,6 +372,9 @@ class Annotation {
       modificationDate: this.modificationDate,
       rect: this.rectangle,
       subtype: params.subtype,
+      actions,
+      fieldName,
+      kidsId,
```

Comment:
This is all exposed in the API now if we add it here, and that's maybe unwanted here given that this is really only interesting for the scripting code and not for annotation API consumers in general?